asllib/TopoSort.mli

```diff
@@ -0,0 +1,81 @@
+(** This modules implement a topological sorting algorithm. *)
```

Comment:
Hum this module implements more than simple "topological sorting". It implements computation of strongly connected components.